LibFont.js

```diff
@@ -219,6 +219,8 @@ const LibFont = (() => {
     glyph = (codepoint) => {
       const index = this.glyphIndex(codepoint);
       const width = this.glyphWidths.at(index);
+      if (!index || width == 0)
```

Comment:
I think this `index` check should be before the `this.glyphWidths.at(index);` and should explicitly be (`index !== null `). 

The width one should also be `width == 0`